modules/custom/social_eda/social_eda.info.yml

```diff
@@ -4,3 +4,4 @@ description: 'Social EDA core module'
 package: Social
 core_version_requirement: ^10 || ^11
 hidden: true
+service_provider: Drupal\social_eda\SocialEdaServiceProvider
```

Comment:
@tregismoreira Why do we need this line? I haven't seen such examples in Drupal Core.